modules/workrelation/model/workrelation.class.php

```diff
@@ -2,8 +2,46 @@
 /* Work relation class
  */
 
+// include validation class
+require_once($_SETUP['HOME_DIR'] . "/code/lib/validation/validation.class.php");
+
 class work_relation {
 
+  public static function validate_furloughs($furloughs) {
+    $errors = array();
+
+    foreach ($furloughs as $furlough) {
+      $id = $furlough->FurloughID;
+      $error_prefix = "Permisjons og permitterings ". $id .": ";
+
+      if(self::empty_field($furlough->FurloughText)) {
+        $errors[] = $error_prefix . "Tekst kan ikke v&aelig;re blank.";
+      }
+
+      if(self::empty_date($furlough->FurloughStart) || !self::valid_date($furlough->FurloughStart)) {
+        $errors[] = $error_prefix . "Start dato kan ikke v&aelig;re blank og m&aring; v&aelig;re en gyldig dato.";
+      }
+
+      if(self::empty_date($furlough->FurloughStop) || !self::valid_date($furlough->FurloughStop)) {
+        $errors[] = $error_prefix . "Slutt dato kan ikke v&aelig;re blank og m&aring; v&aelig;re en gyldig dato.";
+      }
+
+      if(!self::empty_date($furlough->FurloughStop) && $furlough->FurloughStop < $furlough->FurloughStart) {
+        $errors[] = $error_prefix . "Slutt dato kan ikke v&aelig;re f&aring;r start dato.";
+      }
+
+      if(self::empty_percent($furlough->FurloughPercent) || !self::valid_percent($furlough->FurloughPercent)) {
+        $errors[] = $error_prefix . "Prosent kan ikke v&aelig;re tom og m&aring; v&aelig;re st&oslash;rre enn 0 og 100 p&aring; det meste.";
```

Comment:
Should be `Prosent kan ikke v&aelig;re tom og m&aring; v&aelig;re mellom 0 og 100.`